typescript/api/controllers/webservice/RecordController.ts

```diff
@@ -1152,7 +1157,7 @@ export module Controllers {
               if(this.isMetadataEqual(newMetadata,oldMetadata)) {
                 recordResponses.push(response);
```

Comment:
The `response` defined outside the `this.isMetadataEqual(newMetadata,oldMetadata)` check confused me.
Should the response definition be inside this check?

Reply:
Yeah it was a little confusing to me too. I've refactored it.